src/Utilities/Utils.res

```diff
@@ -835,3 +835,5 @@ let isOtherElements = componentType => {
   componentType == "cardExpiry" ||
   componentType == "cardCvc"
 }
+
+let nbsp = `\u00A0`
```

Comment:
why is this needed ? using this is not encouraged

Reply:
This is for adding space in string, if I directly use React.string(` some `), it will trim the spaces and show some